front/src/modules/metadata/utils/formatMetadataFieldAsColumnDefinition.ts

```diff
@@ -35,4 +37,5 @@ export const formatMetadataFieldAsColumnDefinition = ({
   },
   Icon: IconBrandLinkedin,
   isVisible: true,
+  basePathToShowPage: `/object/${metadataObject.nameSingular}/`,
```

Comment:
Maybe we should use the metadataObject's slug?
```suggestion
  basePathToShowPage: `/object/${getObjectSlug(metadataObject)}/`,
```

Reply:
Yes ! I'll implement it in another PR.